testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -403,7 +404,11 @@ def test_plot_mean_profile(self, hole, frames, profiles):
         binned, bins = hole.bin_radii(bins=100)
         mean = np.array(list(map(np.mean, binned)))
         stds = np.array(list(map(np.std, binned)))
-        midpoints = 0.5 * bins[1:] + bins[:-1]
+        midpoints = 0.5 * bins[1:] + 0.5 * bins[:-1]
```

Comment:
```suggestion
        midpoints = 0.5 * (bins[1:] + bins[:-1])
```
Writing it this way, saves a whole bunch of calculations and performance is significantly better:
```python
>>> bins = np.linspace(0, 10, 80)   # typical bins

>>>  %timeit  midpoints = 0.5 * bins[1:] + 0.5 * bins[:-1]
2.15 µs ± 47.1 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

>>> %timeit  midpoints = 0.5 * (bins[1:] + bins[:-1])
1.5 µs ± 6.91 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```
In relative terms, a speed-up of 30% for such a small change is really good.

In practice, the difference between 2.15 µs and 1.5 µs does not matter for the HOLE analysis. But nevertheless we want to write the best code that we can, not the least because existing code serves as an example for other code.